packages/wonder-blocks-core/components/clickable-behavior.js

```diff
@@ -308,25 +308,42 @@ export default class ClickableBehavior extends React.Component<
         } = this.props;
         let shouldNavigate = true;
 
+        // While defaultPrevented is read-only on DOM events, that isn't the
+        // case for SynethicEvents.
+        e.defaultPrevented = false;
```

Comment:
Why do we need to explicitly set this to `false`? Shouldn't the browser be doing that for us?

Reply:
I'll expand on the comment.  The callers of this function call `e.preventDefault()` before we get here so `e.defaultPrevent` is set to `true` by this point.  We want to determine if `onClick(e)` is also calling `e.preventDefault()` so we have to reset the value of `e.defaultPrevent` ourselves before calling `onClick(e)`.

Reply:
I ended up simplify things by not calling `e.preventDefault()` before calling this function.